### ISSUE TITLE
Keep records of deleted ServiceUploads

### ISSUE THREAD

**Issue by alexsoble** [studentinsights/studentinsights#1253]
Right now our app is fully CRUD for ServiceUploads. That's not a good architecture because we always need to have an audit trail for user actions. Instead of removing deleted ServiceUploads from our database, we should record the deleted status on a database field and also log the IDs of the services we deleted. Does that sound about right @kevinrobinson?

**Comment by kevinrobinson** [comment]
@alexsoble Yeah we could handle this by adding fields to the `ServiceUpload` model, or alternately we could make a new model called `Logs` and then just log a JSON object that has the data about this particular action (or any other thing where we want a durable audit trail).

It seems like the second might be simpler for developers: "hey, want to do something sort of destructive?  just log it as JSON!" rather than mixing in this concern in with product features.  If it's needed for the product (like a service being discontinued) then it should go on the model proper but if not just log it (like this case here).

**Comment by alexsoble** [comment]
@kevinrobinson That works for me! `Logs`!
